sonic_platform_base/card_base.py

```diff
@@ -0,0 +1,80 @@
+#
+# card_base.py
+#
+# Abstract base class for implementing a platform-specific class with which
+# to interact with a card in SONiC chassis
+#
+
+class CardBase(object):
+    """
+    Abstract base class for implementing a platform specific class to
+    represent a control-card, line-card or fabric-card of a chassis
+    """
+
+    # Possible card types
+    MODULE_TYPE_CONTROL = "CONTROL-CARD"
+    MODULE_TYPE_LINE    = "LINE-CARD"
+    MODULE_TYPE_FABRIC  = "FABRIC-CARD"
+
+    def get_name(self):
+        """
+        Retrieves the name of the card
```

Comment:
the name here is LINE-CARD/ FABRIC-CARD etc.?
or is this the platform/vendor specified name (i.e. PID (productID))?
If its former, then get_description() reflects the platform/vendor supplied PID?

Can you update the comment section of both these APIs?

Reply:
Yes, get_description() reflects the platform/vendor supplied PID